dev-libs/igraph/files/igraph-0.9.0-cmakedirs.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 723d06d..bb6e041 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -440,8 +440,8 @@ generate_export_header(igraph
+ include(GNUInstallDirs)
+ install(
+   TARGETS igraph
+-  RUNTIME DESTINATION bin
+-  LIBRARY DESTINATION lib
++  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
++  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
```

Comment:
has this been submitted upstream?

Reply:
Am going to send it, they are pretty receptive and chill.

Reply:
igraph developer here. This is fixed in 0.9.1 now, and do let us know if there are any other problems. :-)